packages/govern-contract-utils/contracts/erc20/SafeERC20.sol

```diff
@@ -84,7 +91,6 @@ library SafeERC20 {
                 default { }
             }
         }
-        return ret;
     }
 
     function staticInvoke(address _addr, bytes memory _calldata)
```

Comment:
Not related to this specific PR, so better doing this separately, but we should remove the remaining `staticInvoke()` utilities since the compiler does this automatically now.

Reply:
Sounds good, thanks for the review